modules/pipeline/services/pipelinesvc/run.go

```diff
@@ -133,9 +135,44 @@ func (s *PipelineSvc) RunPipeline(req *apistructs.PipelineRunRequest) (*spec.Pip
 	// send to pipengine reconciler
 	s.engine.Send(p.ID)
 
+	// update pipeline definition
+	if err = s.updatePipelineDefinition(p); err != nil {
+		logrus.Errorf("failed to updatePipelineDefinition,err: %s", err.Error())
```

Comment:
`...o, err: ` pay attention on format please.
and you should print definition id and pipeline id together.

Reply:
done